docs/getting_started.md

```diff
@@ -107,19 +138,19 @@ $ cat /etc/hosts
 
 If you are trying this on your own hosts, then for each host
 1. Download the [byoh-hostagent-linux-amd64](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/releases/download/v0.2.0/byoh-hostagent-linux-amd64)
```

Comment:
Shall we point this to latest release agent ?

Reply:
yes, we should

Reply:
fixed